apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.test.html

```diff
@@ -0,0 +1,2965 @@
+        // FAQ: Design rejected these because each looks like a button
+        // RFE: Change styles to look less like button (done) and request review
+        /*
+			'[inputmode="numeric"]': '#',
+			'input[type="number"]': '#',
+			'input[type="email"]': '@',
+			'input[type="tel"]': '☎',
+			'input[pattern]': '⠿',
+			*/
+      };
+      const inputs = document.querySelectorAll("input");
+
+      [...inputs].forEach((input) => {
+        const selectors = Object.keys(selectorSymbolDict);
+        const matchedSelector = selectors.find((sel) => input.matches(sel));
+        const symbol = selectorSymbolDict[matchedSelector];
+
+        if (symbol) {
+          const prepend = document.createElement("span");
+          const fieldWrapper = input.closest(".field-wrapper");
+
+          prepend.className = "js-prepend--span";
+          prepend.dataset.text = symbol;
+          prepend.innerText = symbol;
+
+          fieldWrapper.classList.add("js-prepend--field-wrapper");
+
+          input.before(prepend);
+          input.classList.add("js-prepend--input");
+        }
+      });
+    </script>
+
+    <style id="form-prepend-input--css">
+      .field-wrapper.js-prepend--field-wrapper {
+        display: grid;
+        align-items: start;
+        grid-template-columns: auto 1fr;
+        grid-template-areas:
+          "label label"
+          "before input"
+          "help help"
+          "error error";
+      }
+      .js-prepend--field-wrapper label {
+        grid-area: label;
+      }
+      .js-prepend--field-wrapper input {
+        grid-area: input;
+      }
+      .js-prepend--field-wrapper .js-prepend--span {
+        grid-area: before;
+      }
+      .js-prepend--field-wrapper .js-append--span {
+        grid-area: after;
+      }
+      .js-prepend--field-wrapper .help-text {
+        grid-area: help;
+      }
+      .js-prepend--field-wrapper .field-errors {
+        grid-area: error;
+      }
+
+      .js-prepend--field-wrapper {
+        --width: 1em;
+        --padding-vert: 5px; /* mimic input */
+        --padding-horz: 10px; /* mimic input */
+      }
+      .js-prepend--span {
+        /* To mimic shape of an input */
+        display: inline-block;
+        padding: var(--padding-vert) var(--padding-horz);
+
+        /* To have consistent size */
+        width: var(--width);
+        text-align: center;
+        box-sizing: content-box;
+
+        /* To limit interaction */
+        user-select: none;
+        pointer-events: none;
+      }
+
+      /* To render <span> atop and (as if) within input */
+      .js-prepend--span {
+        position: relative;
+        top: var(--global-border-width--normal);
+        left: var(--global-border-width--normal);
+        margin-right: calc(-1 * var(--width) - var(--padding-horz) * 2);
+      }
+      .js-prepend--span + input {
+        text-indent: calc(var(--width) + var(--padding-horz));
+      }
+    </style>
+  </aside>
```

Comment:
**Notice.** You should keep this here, for now. Most of it might be removable, but I don't want to rush anyone to pick-and-choose-and-test-again on this hectic day.

<details>

- These are specific to this page.
- Some of these I may modularize (for use by other form-plugin–made forms).
- Most of these manipulate the HTML to the state that it is already at in this form (which was copied from dynamic HTML).
- There is no way (yet) for me to add per-page Snippets to a custom app.
- Having the scripts and styles committed is better than Snippet anyway.

</details>

**Eventually.** After today, I should comb out most of this.